package.json

```diff
@@ -101,7 +101,8 @@
     "wcag": "0.3.0"
   },
   "scripts": {
-    "test": "snyk test"
+    "test": "snyk test",
```

Comment:
Unrelated to this, PR, but this should probably be changed to run our actual test suites.

Reply:
Yeah we have https://github.com/cfpb/cfgov-refresh/issues/3576 and https://github.com/cfpb/cfgov-refresh/issues/2303 that overlap with that.